app/src/main/java/com/hadiyarajesh/notex/ui/note/add/NoteState.kt

```diff
@@ -0,0 +1,22 @@
+package com.hadiyarajesh.notex.ui.note.add
+
+import androidx.compose.runtime.MutableState
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.ui.graphics.toArgb
+import com.hadiyarajesh.notex.ui.theme.*
+
+data class NoteState(
+    val title: MutableState<String> = mutableStateOf(""),
+    val noteDesc: MutableState<String> = mutableStateOf(""),
+    val color: MutableState<Int> =  mutableStateOf(noteColors[0].toArgb())
+){
+    companion object {
+        val noteColors =
+            listOf(White, RedOrange, LightGreen, Violet, BabyBlue, RedPink, LightYellow, LightOrange)
```

Comment:
Can we use Colors provided by compose library?

Reply:
yes we can.....but compose library does not have all the colors I am using . So Imentioned all the colors in colors.kt